lib/dav4rack/xml_response.rb

```diff
@@ -71,35 +68,28 @@ def render_lockdiscovery(*args)
 
 
     # returns an activelock Ox::Element for the given lock data
-    def activelock(time: nil, token:, depth:,
-                      scope: nil, type: nil, owner: nil, root: nil)
-
+    def activelock(time: nil, token:, depth:, scope: nil, type: nil, owner: nil, root: nil)
       Ox::Element.new(D_ACTIVELOCK).tap do |activelock|
         if scope
-          activelock << ox_element(D_LOCKSCOPE, scope)
+          value = Ox::Element.new("#{DAV_NAMESPACE_NAME}:#{scope}")
```

Comment:
It is a good idea to do this in here, but it's a breaking change (I am looking at code that calls this with the namespace prepended already right now). Also has nothing to do with Ruby 3 compatibility and should probably be in a separate PR that can be referenced from the changelog.

I'd opt for adding a check if the namespace is present already.